haste/windowed_conformal_model/conformal_interestingness_model.py

```diff
@@ -125,11 +125,24 @@ def interestingness(self,
             sum_of_intensities_ts_features = time_series_features(sum_of_intensities_ndarray,
                                                                   timestamps_ndarray,
                                                                   timestamp)
+
             # .. = (mean,sd,d1,d2)
 
-            # TODO: concatenate with same ts-series features for correlations
+            #Correlation
```

Comment:
would make more sense to put the correlation first - because thats the order returned from the course_features method